collective/deletepermission/configure.zcml

```diff
@@ -20,6 +20,13 @@
       />
 
   <monkey:patch
+      description="Patch dexterity's manage_delObjects"
+      class="plone.dexterity.content.Container"
+      original="manage_delObjects"
+      replacement="collective.deletepermission.del_object_dexterity.manage_delObjects"
```

Comment:
This imports `del_object_dexterity`, which imports dexterity, which will result in an `ImportError` when dexterity is not installed.
Since we support down to Plone 4.1 we probably have no dexterity installed.

Please add a condition, e.g.:
```xml
<configure
    ...
    xmlns:zcml="http://namespaces.zope.org/zcml"
    >

   <configure zcml:condition="installed plone.app.dexterity">
      <monkey:patch ... />
   </configure>
```